crates/astria-sequencer/src/service/info/mod.rs

```diff
@@ -60,8 +60,8 @@ impl Info {
             .context("invalid path: `asset/denom/:id`")?;
         query_router
             .insert(
-                "asset/allowed_fee_asset_ids",
-                crate::asset::query::allowed_fee_asset_ids_request,
+                "asset/allowed_fee_assets",
```

Comment:
To confirm our offline discussion: this change is correct, just overlooked this in the big refactor to remove IDs